app/locales/pt-BR.json

```diff
@@ -0,0 +1,118 @@
+{
+  "label": {
+    "about_title": "Sobre",
+    "authorities_add_button_label": "Adicionar uma fonte confiável",
+    "authorities_add_url": "Adicionar uma autoridade via URL",
+    "authorities_desc": "Escolha autoridades de saúde confiáveis ​​em sua área para obter dados de exposição. Selecione um nome no registro global, ou digite o endereço da web fornecido por uma autoridade que implementou o Safe Paths.",
+    "authorities_input_placeholder": "Cole a URL aqui",
+    "authorities_no_sources": "Nenhuma fonte de dados ainda",
+    "authorities_removal_alert_cancel": "Cancelar",
+    "authorities_removal_alert_desc": "Você tem certeza que deseja remover this authority data source?",
+    "authorities_removal_alert_proceed": "Continuar",
+    "authorities_removal_alert_title": "Remover autoridade",
+    "authorities_title": "Fontes Confiáveis",
+    "choose_provider_subtitle": "Para ser informado de exposição você deverá contatar uma agência de saúde.",
+    "choose_provider_title": "Escolha a autoridade de saúde",
+    "commitment": "Comprometimento",
+    "commitment_para": "Safe Paths armazena e checa sua interação com as pessoas usando sua localização. Os seus dados NUNCA sairão dos seu celular sem o seu consentimento.",
```

Comment:
What do you think of this?
Safe Paths verifica e grava sua interação com outras pessoas de forma segura utilizando sua localização.

It emphasizes the security in checking and recording your location in the first sentence.

Reply:
You're right! Thanks for the suggestion.